recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -22,10 +23,20 @@ if "%CONDA_BLD_PATH%" == "" (
 )
 
 :: Increase pagefile size, cf. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/155
+:: Both in the recipe and in the final package, this script is co-located with
+:: SetPageFileSize.ps1 & ExecuteSetPageFileSize.bat, see meta.yaml
+set ThisScriptsDirectory=%~dp0
+set EntryPointPath=%ThisScriptsDirectory%ExecuteSetPageFileSize.bat
 if "%CI%" == "azure" (
+    REM use different drive than CONDA_BLD_PATH-location for pagefile
     if "%CONDA_BLD_PATH%" == "C:\\bld\\" (
-        REM this file will be run within %SCRIPTS%, where SetPageFileSize.ps1 is also copied into, see meta.yaml
-        call SetPageFileSize.ps1 -MinimumSize 4GB -MaximumSize 8GB -DiskRoot "D:"
+        echo CONDA_BLD_PATH=%CONDA_BLD_PATH%; Setting pagefile size to 8GB on D:
+        REM Arguments are -MinimumSize, -MaximumSize, -DiskRoot see ExecuteSetPageFileSize.bat
+        call %EntryPointPath% 8GB 8GB D:
```

Comment:
Why go with an indirection when it can just be
```suggestion
        PowerShell -NoProfile -ExecutionPolicy Bypass -Command "& '%EntryPointPath%' -MinimumSize 8GB -MaximumSize 8GB -DiskRoot D:"
```

Reply:
Thanks makes sense. I just followed the blog you posted very closley because windows never ceases to amaze with trivial assumptions that end up _not_ working.